openhtf/io/proto/testrun.proto

```diff
@@ -0,0 +1,257 @@
+// Copyright 2014 Google Inc. All Rights Reserved.
```

Comment:
heh, lots of files have a 2014 date, now that I look at it.. updated here since we're adding it, maybe if i feel ambitious i'll do another PR updating all the copyright notices.